core/templates/pages/exploration-player-page/learner-experience/tutor-card.directive.ts

```diff
@@ -219,9 +219,6 @@ angular.module('oppia').directive('tutorCard', [
               UrlInterpolationService.getStaticImageUrl(
                 '/avatar/oppia_avatar_100px.svg'));
 
-            $scope.profilePicture = UrlInterpolationService.getStaticImageUrl(
```

Comment:
I don't see corresponding changes in `tutor-card.directive.html`.

Reply:
there is no usage of this variable in the tutor-card.directive.html file

Reply:
Looks like it is used here:

https://github.com/darksun27/oppia/blob/77dc7ae826d6d34e00c4eb76e6482ee4dcd828e8/core/templates/pages/exploration-player-page/learner-experience/tutor-card.directive.html#L42-L47

and here

https://github.com/darksun27/oppia/blob/77dc7ae826d6d34e00c4eb76e6482ee4dcd828e8/core/templates/pages/exploration-player-page/learner-experience/tutor-card.directive.html#L54-L59

Reply:
On that note, how have you tested your changes?

Reply:
@darksun27 PTAL!

Reply:
Tested the changes using the tests given. I didn't realise that the variable was used outside an image tag. Sorry for the mistake. Corrected the changes and pushing the code back.

Reply:
@kevintab95 PTAL!

Reply:
Hi @darksun27, can you explain what you mean by "Tested the changes using the tests given".  I'm not sure we have tests to check if images are loaded -- I could be wrong. Can you please check if the images you have added work as expected (manually if possible)? There may be cases like this that may have been missed. This can lead to having broken images on the app and that is a serious regression.

Reply:
Ohh, for images being loaded checked by running the application. Will provide screenshots for the same.

Reply:
Sounds good. Thanks!

Reply:
![On About Page](https://user-images.githubusercontent.com/31998550/84055994-3a540580-a9d3-11ea-869f-e541a774fb76.png)
![Oppia Logo](https://user-images.githubusercontent.com/31998550/84056030-463fc780-a9d3-11ea-8b7c-dcc8e75b539f.png)

Reply:
@kevintab95 PTAL!

Reply:
Thanks, just to confirm, have you manually verified this for all images you have introduced? Have you also checked that it fallsback to png version of files if the browser does not support webp?

Reply:
Yes I have checked the most of the images. Some images were empty in both the formats PNG as well as WebP [Eg: about-page-mascot]. Not tested for the fallback on purpose but while testing I saw PNG images were being requested.

Reply:
> Not tested for the fallback on purpose but while testing I saw PNG images were being requested.

I'm not sure I understand. Do you mean PNG is requested when WEBP is not supported?

Reply:
Yes

Reply:
@kevintab95 I have checked most of the file for loading, But I am not able to get to the skill-editor page and the param editor page to check whether the drag-dots are visible or not. Can you help me reach that state?
![Screenshot from 2020-06-09 13-11-34](https://user-images.githubusercontent.com/31998550/84127057-5b146d80-aa5c-11ea-94c2-437a1232a890.png)
![Screenshot from 2020-06-09 13-11-48](https://user-images.githubusercontent.com/31998550/84127062-5c459a80-aa5c-11ea-911f-177ea7c12013.png)
![Screenshot from 2020-06-09 13-20-45](https://user-images.githubusercontent.com/31998550/84127064-5cde3100-aa5c-11ea-9c76-d119e9d84c50.png)

Reply:
Hi @darksun27, to access the skill editor, you will need to grant admin access to your username (login as admin >  roles > grant admin role to your user name). After that you can access the topics and skills dashboard using the dropdown in the top right of your screen.
![image](https://user-images.githubusercontent.com/11008603/84133849-50120b00-aa65-11ea-8d9a-906087b3d830.png)


For param editor you will need to create and exploration and in the setting tab, enable parameters.
![image](https://user-images.githubusercontent.com/11008603/84134008-8cde0200-aa65-11ea-8195-9f441c47fe11.png)

After that you'll see the parameter editor in the main editor.
![image](https://user-images.githubusercontent.com/11008603/84134083-ada65780-aa65-11ea-81e7-a36a3e1e46fb.png)

Reply:
Thanks a lot for the help @kevintab95

Reply:
![Screenshot from 2020-06-09 16-42-57](https://user-images.githubusercontent.com/31998550/84142210-0380fc80-aa72-11ea-9a11-89b9472c39a5.png)
![Screenshot from 2020-06-09 16-48-17](https://user-images.githubusercontent.com/31998550/84142214-054ac000-aa72-11ea-9821-e83876275cba.png)